### PR TITLE
feat: [UX] Permalink to event from build detail page

### PR DESCRIPTION
**Pull request by OronNadiv**
## Context
Users would like to navigate from the build details page to the pipeline while highlighting a selected job associated with the build.

## Objective

Users can click on a link in the build details page that would redirect them back to the pipeline while highlighting a selected job associated with the build.

image

image

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by tkyi** [comment]
Can you post a screenshot of what the new change would look like compared to the old?

**Comment by OronNadiv** [comment]
@tkyi done.  See PR description.

**Comment by tkyi** [comment]
Wonder if something more explicit like `< Back to event` in the upper left or breadcrumbs like in commands UI (e.g.: https://cd.screwdriver.cd/commands/screwdriver/docker-trigger) would be more clear

But okay with doing it this way until we have a more fleshed out design by @chasturansky 👍

**Comment by tkyi** [comment]
Looks like a few tests are failing

File: app/pipeline/events/controller.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ import ModelReloaderMixin from 'screwdriver-ui/mixins/model-reloader';
 import { isPRJob } from 'screwdriver-ui/utils/build';
 
 export default Controller.extend(ModelReloaderMixin, {
+  queryParams: [
+    {
+      selectedJobId: { type: 'string' }
```
Comment (adong) [inline app/pipeline/events/controller.js:14]: Cool, "Explicit is better than implicit."  [queryParams](https://api.emberjs.com/ember/3.21/classes/Controller/properties/queryParams?anchor=queryParams)

**Comment by OronNadiv** [comment]
> Looks like a few tests are failing

@tkyi Yes, they now pass.

**Review by tkyi — APPROVED** [review]

**Pull request merged**